### DISCUSSION TITLE
Release circe-yaml v0.15.0

### DISCUSSION THREAD

**Discussion by jeffmay** [circe/circe-yaml#363] [General]
This release contains the upgrade from Snake YAML 1.33 to 2.0 which is a binary compatibility breaking major upgrade.

This discussion is a placeholder for anyone who is using v0.15.0-RC1 to comment or upvote this discussion for the promotion of this RC version to v0.15.0

Thank you!

_Tagging folks who contributed to the changes in this version from v0.14.3:_
@sideeffffect @jkobejs

**Comment by sideeffffect** [comment]
Just to be super clear, Snake YAML 2.0 is still YAML 1.1 (not 1.2), correct?

**Reply by guizmaii** [reply]
AFAIU, `snake-engine` is the YAML 1.2 implementation of snakeyaml

**Comment by mziolekda** [comment]
We have given the new version a whirl in the daml project and it worked just fine. So up-voting the release.

**Comment by SpyQuel** [comment]
Up-voting this release!

**Comment by rciccone91** [comment]
I've updated this version in our project and it's working flawlessly. Up-voting for this release!

**Comment by arshadusama** [comment]
Up-voting this release!

**Comment by emanuelemaffeo** [comment]
Up-voting this release!

**Comment by jeffmay** [comment]
So sorry for the delay on this, y'all. I no longer work with Scala and I get a ton of github notifications at work. I recently saw a question about this release and came back around to finish it.

https://github.com/circe/circe-yaml/releases/tag/v0.15.0
